tests/parm/postxconfig-NT_FH00.txt

```diff
@@ -1,6 +1,6 @@
 2
 47
-150
+162
```

Comment:
@ericaligo-NOAA I would suggest you use parm/postxconfig-NT-GFS-F00-TWO.txt from the UPP develop branch to update this file.